modules/keycloak/variables.tf

```diff
@@ -27,4 +27,6 @@ variable "frontend_url" {}
 variable "ip_whitelist" {
   description = "IP addresses allowed to access"
   default     = ["0.0.0.0/0"]
-}
\ No newline at end of file
+}
+
+variable "kms_key_id" {}
```

Comment:
Could you just stick a description in here so it's clear it's used as the database encryption key?

Reply:
Yep, I've added one in.